endpoints.json

```diff
@@ -28,5 +28,8 @@
   },
   "GET /api/articles/:article_id": {
     "description": "serves an array of an article found by article_id"
+  },
+  "GET /api/articles/:article_id/comments": {
+    "description": "serves an array of comments for an article with a specific article_id"
```

Comment:
Where is the rest of this? This file should contain much more information per end point

Reply:
More information added in endpoint.json file accordingly.